rif-collector-app.php

```diff
@@ -34,4 +34,92 @@ function retrieveAllQuery($db) {
     return $query->fetchAll();
 }
 
-?>
+$message = "";
+$validatedSanitizedArr = [];
```

Comment:
No floating variables in function files please - they will get initialised in every file that requires this file - you don't want to pull in variables too if all you want is a function from the file!